### PR TITLE
Feature/school 759/add button for adding circles

### PR DESCRIPTION
**Pull request by LeonidErshov**

### CODE REVIEW

File: apps/schools/domains/common/components/input/index.tsx [hunk 1]
```diff
@@ -63,6 +71,7 @@ export const Input: React.FC<CustomInputProps> = (props) => {
                     buttonStyle={BUTTON_INPUT_PHONE_STYLE}
                     inputStyle={INPUT_PHONE_STYLE}
                     containerStyle={{ marginTop: '8px' }}
+                    value={phoneValue}
```
Comment (mzavarka) [inline apps/schools/domains/common/components/input/index.tsx:74]: we should test this logic

File: apps/schools/domains/common/components/table/interfaces.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { ReturnedData } from '@domains/common/redux/interfaces'
 import { TableProps } from 'antd'
 import { ColumnFilterItem } from 'antd/lib/table/interface'
-import { Key } from 'antd/es/table/interface'
+import {Key} from "antd/es/table/interface";
```
Comment (mzavarka) [inline apps/schools/domains/common/components/table/interfaces.ts]: strange, did you configure autolinter?

**Review by mzavarka — CHANGES_REQUESTED** [review]

File: apps/schools/domains/common/components/table/interfaces.ts (hunk 2 above)
Comment (LeonidErshov) [reply apps/schools/domains/common/components/table/interfaces.ts]: Change it

File: apps/schools/domains/common/components/input/index.tsx (hunk 1 above)
Comment (LeonidErshov) [reply apps/schools/domains/common/components/input/index.tsx:74]: Tested

**Pull request merged**